run_functional_tests.sh

```diff
@@ -22,42 +15,35 @@ fi
 ELEVATE_PRIVILEGES=${ELEVATE_PRIVILEGES:-}
 
 if [ "$DOCKER" = "1" ]; then
-    CCF_HOST=$CCF_HOST CCF_PORT=$CCF_PORT \
-        ./docker/run-dev.sh &
+    export CCF_HOST=${CCF_HOST:-"localhost"}
+    export CCF_PORT=${CCF_PORT:-8000}
+    export CCF_URL="https://${CCF_HOST}:${CCF_PORT}"
+
+    ./docker/run-dev.sh &
+    CCF_NETWORK_PID=$!
+    trap "kill $CCF_NETWORK_PID" EXIT
+
+    # Turn off pytest output capture to allow test logs to be interleaved with
+    # the docker logs.
+    TEST_ARGS="-s"
 else
-    ./start.sh &
+    SCITT_DIR=/tmp/scitt
+    TEST_ARGS="--start-cchost --enclave-type=$ENCLAVE_TYPE --enclave-package=$SCITT_DIR/lib/libscitt --constitution=$SCITT_DIR/share/scitt/constitution"
 fi
 
-CCF_NETWORK_PID=$!
-trap "kill $CCF_NETWORK_PID" EXIT
-
-# wait until the network is ready
-timeout=120
-while ! curl -s -f -k $CCF_URL/app/parameters > /dev/null; do
-    echo "Waiting for service to be ready..."
-    sleep 1
-    timeout=$((timeout - 1))
-    if [ $timeout -eq 0 ]; then
-        echo "Service failed to become ready, exiting"
-        exit 1
-    fi
-done
```

Comment:
Isn't this still needed for the docker case?

Reply:
It does, but that's running in the background, so there's a race here.

Reply:
Yes you're right. Fixed it now.